servicetalk-grpc-netty/src/main/java/io/servicetalk/grpc/netty/GrpcClients.java

```diff
@@ -90,6 +91,41 @@ public static GrpcClientBuilder<HostAndPort, InetSocketAddress> forAddress(final
         return applyProviders(address, new DefaultGrpcClientBuilder<>(() -> HttpClients.forSingleAddress(address)));
     }
 
+    /**
+     * Creates a {@link GrpcClientBuilder} for an address with default {@link LoadBalancer} and DNS
+     * {@link ServiceDiscoverer} using the specified {@link DiscoveryStrategy}.
+     * <p>
+     * The returned builder can be customized using {@link GrpcClientBuilderProvider}.
+     *
+     * @param host host to connect to, resolved by default using a DNS {@link ServiceDiscoverer}.
+     * @param port port to connect to
+     * @param discoveryStrategy {@link DiscoveryStrategy} to use
+     * @return new builder for the address
+     * @see GrpcClientBuilderProvider
+     */
+    public static GrpcClientBuilder<HostAndPort, InetSocketAddress> forAddress(
+            final String host, final int port, final DiscoveryStrategy discoveryStrategy) {
```

Comment:
Consider moving the `DiscoveryStrategy` to a namespace where it is not tied to http or grpc, since it could be technically also used for other protocols?